pkg/virt-controller/watch/migration.go

```diff
@@ -399,7 +403,21 @@ func (c *MigrationController) updateStatus(migration *virtv1.VirtualMachineInsta
 			}
 		case virtv1.MigrationPending:
 			if podExists {
-				migrationCopy.Status.Phase = virtv1.MigrationScheduling
+				if controller.VMIHasHotplugVolumes(vmi) {
+					attachmentPods, err := controller.AttachmentPods(pod, c.podInformer)
+					if err != nil {
+						return fmt.Errorf("failed to get attachment pods: %v", err)
+					}
+					if len(attachmentPods) > 0 {
+						attachmentPod := attachmentPods[0]
+						if isPodReady(attachmentPod) {
```

Comment:
I'd rather the pod "ready" condition for both the target and attachment pod be an indication that the migration is "scheduled"

Here we should just be checking that the attachment pod exists.